src/snap/actions/types.d.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Silence Laboratories Pte. Ltd.
+// This software is licensed under the Silence Laboratories License Agreement.
+
+export interface PairingDataInit {
+  pairingId: string;
+  encPair: _sodium.KeyPair;
+  signPair: _sodium.KeyPair;
+}
+
+export interface IPairingAction {
```

Comment:
I don't think we need to have interfaces for all the classes, we could use classes as types. Even in mock test, JS doesn't require them.